src/markup.c

```diff
@@ -64,7 +64,7 @@ static char *markup_strip_a(char *str)
                 // If we hit the end of the string, delete the trailing tag fully
                 unsigned int len = end ? (end-start)+1 : strlen(start);
 
-                str = string_replace_at(str, start-str, len, "");
+                str = string_replace_at(str, start-str, len, "<span color=\"blue\"><u>");
```

Comment:
While it's specified by the spec, I can already see all the customization requests a mile away, especially since it looks ugly even with the default theme.

![2017-11-29t18-31-03](https://user-images.githubusercontent.com/18189112/33386364-7a00aac0-d533-11e7-8d45-404d9466765e.png)

Reply:
Yes, this *is* ugly. I just pushed it as an example how the ugly the recommendation will look like, when we discussed, what the replacement for the link replacement method.

What would you like to do? Remove it fully and mention it to use the indicators? Make it more sensible by just underlining it?

Reply:
I would say remove it for now and file an issue about it to look into it later.